spec/models/allocation_spec.rb

```diff
@@ -128,4 +128,21 @@
       end
     end
   end
+
+  describe '#hourly_rate_currencies' do
+    let!(:brl_allocations) { create_list(:allocation, 2, hourly_rate_currency: 'BRL') }
+    let!(:usd_allocation) { create(:allocation, hourly_rate_currency: 'USD') }
+
+    subject { described_class.hourly_rate_currencies }
+
+
+    it 'gets the available currencies from allocations without duplicates in ASC order' do
+      expect(subject).to eq (
+        [ 
+          usd_allocation.hourly_rate_currency,
```

Comment:
It's always good to avoid as much as possible any logic on expectations. In this case, since you know exactly what you are expecting based on your setup, you could do this way: `expect(subject).to eq(['USD', 'BRL'])`